lms/templates/instructor/instructor_dashboard_2/membership.html

```diff
@@ -33,7 +33,7 @@
 </script>
 
 <div class="batch-enrollment membership-section">
-  <h2> ${_("Batch Enrollment")} </h2>
+  <h3 class="hd hd-2">${_("Batch Enrollment")}</h3>
```

Comment:
Hmm that sounds like a lot of scope creep to me. I'd fix it here and mark a TODO for adjusting the style once it's been adjusted.